internal/inventory/inventory.go

```diff
@@ -0,0 +1,13 @@
+package inventory
+
+import (
+	"kg/procurement/internal/vendors"
+)
+
+type Item struct {
+	ID          string         `json:"id"`
+	Name        string         `json:"name"`
+	Description string         `json:"description"`
+	Vendor      vendors.Vendor `json:"vendor"`
```

Comment:
for the model, we could just use `VendorID` for now
because we wouldn't want to populate the vendor on inventory retrieval since that would breach the `Single Responsibility Principle`

Reply:
How is this related to SRP? I think using `vendors.Vendor` would make it easier to test

Reply:
> How is this related to SRP? I think using `vendors.Vendor` would make it easier to test

because on populating during accessor fetch, we would want to populate each field on the `Inventory` struct
if we use the `Vendor` model for the field, we would need to fetch the data again from the database

One might argue, you could do a `JOIN` on the inventory accessor, but again, back to the SRP concept of why are you accessing vendor as well instead of just inventory

Reply:
So are you implying that an accessor should only be concerned with its domain? For example, the inventory accessor should only retrieve its items and not items related to them.

Reply:
Alright then, I agree that we could just use `VendorID`

Reply:
I also agree that we should just use VendorID since we don't use any ORM too to handle this kinds of things.